gensim/corpora/lowcorpus.py

```diff
@@ -166,8 +166,8 @@ def save_corpus(fname, corpus, id2word=None, metadata=False):
 
         if truncated:
             logger.warning("List-of-words format can only save vectors with "
-                            "integer elements; %i float entries were truncated to integer value" %
-                            truncated)
+                           "integer elements; %i float entries were truncated to integer value" %
```

Comment:
Dtto (please use hanging indent consistently).

Also, we want to be passing logging message string parameters as arguments, not using them to format the string directly (wasted effort if message doesn't pass logging filters): `logger.fnc("%s", x)`, not `logger.fnc("%s" % x)`.